src/analysis/bounded_variation.lean

```diff
@@ -520,6 +530,96 @@ begin
   rw [← evariation_on.union f A B, ← inter_union_distrib_left, Icc_union_Icc_eq_Icc hab hbc],
 end
 
+/-TODO : where should the following two lemmas go? -/
```

Comment:
does it fit in the file defining `monotone_on`?

Reply:
Maybe at the bottom of the file because I don't really see it belonging anywhere particular. Problem is it involves mathlib4 sync'ing, but I'm getting used to it :)

Reply:
It should actually go in `data.set.function` because of the import chain.

Reply:
It's now at the bottom of that file.